docs/labs/EclipseLoot.md

```diff
@@ -94,7 +98,12 @@ It's smart about what parameters are required.
   but leaves its directory, files, etc.
   intact out in the filesystem.
   I would recommend you close instead of remove.
-
+  
+    ![Removing_EclipseLoot](https://github.com/comp129/comp55/assets/148159945/1848723a-3e5e-4e42-9fe9-4e14ded1e3fe)
+  
+        Figure: Removing projects from Eclipse is a key way to keep one's workspace clean and helps with navigating through
+          Package Explorer without scrolling to find projects, classes, etc. In addition, we recommend to not "Delete Content
+          From Disk" since it helps keep one's workspace clean without the nerve of losing progress when it is located in your filesystem.
```

Comment:
```suggestion
        > Removing projects from Eclipse is a key way to keep one's workspace clean and helps with navigating through
        > Package Explorer without scrolling to find projects, classes, etc.
        > In addition, we recommend to ***uncheck*** the option *Delete Content From Disk*
        > since leaving the project helps keep one's workspace clean without risking losing any progress since it stays on your computer.
```